content/material/2019-11-15-intro-dplyr.Rmd

```diff
@@ -49,14 +49,16 @@ install.packages("dplyr")
 We'll be using a very simple dataset that is built-in to the tidyr package. This dataset contains data on tuberculosis cases in Afghanistan, Brazil, and China from 1999 and 2000.
 
 ```{r}
-library(dplyr)
+library(tidyverse)
```

Comment:
```suggestion
# Or library(tidyverse)
library(dplyr)
```

Travis might throw an error, since tidyverse isn't installed on it.

Reply:
Travis is the build system to create the website. It is like a clean computer with only the essential things installed. Tidyverse is big and I haven't installed it on it.

Reply:
I've made a "suggestion" above. Go through all the comments I've made and click the "Add suggestion to batch". Then when you've added all my suggestions, click the "commit suggestions" which should be on the top of the page, near the "Review" green button.

Reply:
When I ran the class the dataframe would not load if only dplyr was loaded, that is why I wrote library(tidyverse)

Reply:
Also, no idea where and how I am supposed to integrate your comments

Reply:
Adding to batch can only be done when you go to the "Files changed" tab at the top of this page. Where is says "Conversation", "Commits", etc

Reply:
And ok, right, you'll have to also load in tidyr, since those datasets come from tidyr.